src/interfaces/logs.ts

```diff
@@ -22,6 +22,8 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
+import { Method } from 'axios'
```

Comment:
![image](https://user-images.githubusercontent.com/19143644/127270794-aa4da35a-eb3b-4654-a9b7-9234efdfdba5.png)

Axios method doesn't have purge, link, and unlink.

Reply:
Related to other comment, the types are defined like this:
![image](https://user-images.githubusercontent.com/16670475/127275020-c0d7ed85-3d9c-49ca-bc78-39e55959f1a2.png)

I posted the filename just in case you are wondering